### PR TITLE
feat(addon-editor): support customization font-size

### LINKED ISSUE #2109: 🐞 - После обновления taiga-ui/addon-editor генерируется другой html для заголовков из-за чего слетает вёрстка
### Which @taiga-ui/* package(s) are the source of the bug?

addon-editor

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/components/editor-new и https://taiga-ui.dev/components/editor

### Is this issue blocking you?

Non-Blocking

### Description

[Старая реализация](https://taiga-ui.dev/components/editor), где 5 видов заголовков, генерируется тег `<font size = "n">test123</font>`
[Новая реализация](https://taiga-ui.dev/components/editor-new), где 3 вида заголовков, генерируются теги `<h1>test123</h1>, <h2>test123</h2> и <p>test123</p>`
При обновлении со старой версии (2.47.0) на новую (2.50.2) слетает вёрстка.

### Angular version

11.2.11

### Taiga UI version

2.47.0 и 2.50.2

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2109

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts [hunk 1]
```diff
@@ -19,10 +19,18 @@ export class TuiFontSizeComponent {
...
+                    px: 13,
+                    name: 'Small',
+                },
                 {
                     px: 15,
                     name: texts.normal,
                 },
+                {
+                    px: 17,
+                    name: 'Large',
```
Comment (nsbarsukov) [inline projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts:32]: i18n ?

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts [hunk 2]
```diff
@@ -52,11 +60,11 @@ export class TuiFontSizeComponent {
         >,
     ) {}
 
-    onClick({headingLevel}: Partial<TuiEditorFontOption>): void {
+    onClick({headingLevel, px}: Partial<TuiEditorFontOption>): void {
         if (headingLevel) {
             this.editor.setHeading(headingLevel);
         } else {
-            this.editor.setParagraph();
+            this.editor.setParagraph({fontSize: `${px}px`});
```
Comment (nsbarsukov) [inline projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts:67]: use `import {tuiPx} from '@taiga-ui/cdk';`

Comment (splincode) [reply projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts]: fixed

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-editor/extensions/font-size.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    addGlobalAttributes() {
+        return [
+            {
+                types: this.options.types,
+                attributes: {
+                    fontSize: {
+                        default: null,
+                        parseHTML: element => element.style.fontSize,
+                        renderHTML: attributes => {
```
Comment (waterplea) [inline projects/addon-editor/extensions/font-size.ts:41]: Destruct :)

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts [hunk 4]
```diff
@@ -16,13 +17,23 @@ import {map} from 'rxjs/operators';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TuiFontSizeComponent {
+    readonly normalFontSize = 15;
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts:20]: Maybe move it to a constant or DI token? Overall, these font options don't seem to be customizable, right?

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.template.html [hunk 5]
```diff
@@ -22,6 +22,7 @@
                 *ngFor="let item of fontsOptions$ | async"
                 tuiOption
                 [style.fontFamily]="item.family"
+                [attr.data-name]="item.name"
```
Comment (waterplea) [inline projects/addon-editor/components/toolbar-tools/font-size/font-size.template.html:25]: What do we need this for?

Comment (splincode) [reply projects/addon-editor/components/toolbar-tools/font-size/font-size.template.html]: fixed

File: projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts (hunk 4 above)
Comment (splincode) [reply projects/addon-editor/components/toolbar-tools/font-size/font-size.component.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**